index.js

```diff
@@ -0,0 +1,26 @@
+var fs = require('fs');
+
+function getEngine(engineName) {
+  var runtime = require('./lib/' + engineName);
+  var sourceBundle = fs.readFileSync(require.resolve('./lib/' + engineName + '/bundle'), 'utf8');
+  var pathToCompiler = require.resolve('./lib/compiler');
+  var Compiler = require(pathToCompiler)(runtime, sourceBundle);
```

Comment:
Why not just use `require('./lib/compiler')`? And require it outside of the `getEngine` function?